Formula/gotop.rb

```diff
@@ -1,14 +1,14 @@
 class Gotop < Formula
   desc "Terminal based graphical activity monitor inspired by gtop and vtop"
-  homepage "https://github.com/cjbassi/gotop"
-  url "https://github.com/cjbassi/gotop/archive/3.0.0.tar.gz"
-  sha256 "d5147080bb6057f0bf0900b38438e89aa066959c845bdd4c84a9c9fe478b176f"
+  homepage "https://github.com/xxxserxxx/gotop"
+  url "https://github.com/xxxserxxx/gotop/archive/3.3.0.tar.gz"
+  sha256 ""
```

Comment:
This needs a sha256 hash. Download the file at `url` and run `sha256sum` on it.

Reply:
Alright, will do – please see my reply to your prior comment as well. 🙂